gradoop-temporal/src/main/java/org/gradoop/temporal/util/TimeFormatConversion.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.temporal.util;
+
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+
+/**
+ * Utility class that provides methods for the conversion of different time formats.
+ */
+public class TimeFormatConversion {
+
+    /**
+     * Converts a {@link LocalDateTime} object and converts it to respective milliseconds since Unix Epoch.
+     * The assumed time zone is UTC.
+     *
+     * @param time time stamp to be converted to milliseconds since Unix Epoch.
+     * @return Representation of the the provided time stamp in milliseconds since Unix Epoch.
+     */
+  public static long convertLocalDateTimeToEpochMilli(LocalDateTime time) {
```

Comment:
I think we can shorten the function name, e.g. `toEpochMilli(LocalDateTime time)` because the class name is already descriptive.